### PR TITLE
Add prominent Extensions link in Download page

### PR DESCRIPTION
**Pull request by AtesComp**
In the Download page, add an Extensions section below the 'Other platforms and versions' to make the Extensions more visible to users.

Also, update the RDF Transform extension text in the Extensions page.

### CODE REVIEW

**Comment by wetneb** [comment]
See the preview at https://deploy-preview-351--openrefine-website.netlify.app/download, towards the bottom.

File: src/pages/download.tsx [hunk 1]
```diff
@@ -87,6 +87,12 @@ function MainDownload() {
         <p>See the <a href="https://github.com/OpenRefine/OpenRefine/releases">full list of releases on GitHub</a> for other versions.</p>
         </div>
       </div>
+      <div className="row" style={{maxWidth: '800px', margin: 'auto' }}>
+        <div className="col">
+        <h3 id="extensions">Extensions</h3>
+        <p>See the <a href="/extensions">extensions</a> for additional ways to use and extent OpenRefine.</p>
```
Comment (wetneb) [inline src/pages/download.tsx:93]:
```suggestion
        <p>See the <a href="/extensions">extensions</a> for additional ways to use and extend OpenRefine.</p>
```

**Comment by thadguidry** [comment]
Looks good!

Side Note:  Gosh I just noticed that those blue links are so close to black and almost make the links invisible depending on how large the font sizing is across the site.  Hmm... I think we might make 1 more tweak to `<a>` link colors with just a bump up in their brightness.  What do you think @wetneb ?  I can do a PR to tweak and review tomorrow or next day.

**Comment by magdmartin** [comment]
looks good to me.  I will merge the PR tomorrow if there are no other comments.

**Comment by magdmartin** [comment]
thanks for catching the typo @wetneb

**Pull request merged**